modules/4_nodes/nodes.tf

```diff
@@ -186,9 +196,9 @@ resource "null_resource" "remove_worker" {
     when       = destroy
     on_failure = continue
     inline = [<<EOF
-oc adm cordon worker-${count.index}
-oc adm drain worker-${count.index} --force --delete-local-data --ignore-daemonsets --timeout=180s
-oc delete node worker-${count.index}
+oc adm cordon ${self.triggers.node_prefix}-worker-${count.index}.${self.triggers.cluster_id}.${self.triggers.cluster_domain}
```

Comment:
Is it required to use the domain here? would not it get resolved without the domain part?

Reply:
It requires the node name, `eu-de-2-master-0.ocp4.cloud.lab` is node name, `eu-de-2-master-0` is not.

Reply:
`node_prefix` will have a `-` at the end from ocp.tf right?